common/chains/client/models.go

```diff
@@ -15,3 +19,21 @@ const (
 	ExceedsMaxFee                            // Attempt's fee was higher than the node's limit and got rejected.
 	FeeOutOfValidRange                       // This error is returned when we use a fee price suggested from an RPC, but the network rejects the attempt due to an invalid range(mostly used by L2 chains). Retry by requesting a new suggested fee price.
 )
+
+type NodeTier int
```

Comment:
Does this need to be exported?

Reply:
Yes, they are used by the `rpc_client.go` file